3.5/s2i/bin/run

```diff
@@ -15,6 +15,12 @@ function should_migrate() {
   is_django_installed && [[ -z "$DISABLE_MIGRATE" ]]
 }
 
+# Utility function to update PYTHONPATH and change cwd
+function change_cwd() {
+  export PYTHONPATH=${PYTHONPATH:-}:`pwd`
```

Comment:
Right, thanks for spotting this. Although the problematic colon found your way into your version as well. ``` export PYTHONPATH=${PYTHONPATH:+$PYTHONPATH:}`pwd` ``` seems to be what we want.